src/main/resources/templates/db_scripts/schema.json

```diff
@@ -21,6 +21,13 @@
       "fromModuleVersion": 1.0,
       "withMetadata": true,
       "populateJsonWithId": true
+    },
+    {
+      "tableName": "voucher_lines",
+      "generateId": true,
```

Comment:
According to [Raml-Module-Builder - The Post Tenant API](https://github.com/folio-org/raml-module-builder#the-post-tenant-api):
> `generateId` - No longer supported. <...> The solution is to generate a UUID in java

Reply:
I was aware of this but I saw that it is not released yet, and is just in the master,  and there are other places where we have to change this. I can make the changes here or deal with all of the changes at once

Reply:
Yes going forward we shouldn't be using this